bot.py

```diff
@@ -81,8 +84,8 @@ async def on_reaction_remove(reaction, user):
     if str(reaction.emoji) == EMOJI_DISPLAY and message.id in starboard_messages:
         await update_title(reaction.count, message)
         starboard_message = await STARBOARD_CHANNEL.fetch_message(STARBOARD_MESSAGE_ID)
-        embed = starboard_message.embeds[0]
-        await starboard_message.edit(content=TITLE, embed=embed)
+        embed = starboard_message.embeds
+        await starboard_message.edit(content=TITLE, embeds=embed)
```

Comment:
this if body is the same as the updating case for on_reaction_add at line 74, can it be deduplicated?